contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -26,6 +26,52 @@
 WIRE_GOSSIP_STORE_CHANNEL_AMOUNT = 4101
 
 
+class LnFeatureBits(object):
+    """ feature flags taken from bolts.git/09-features.md
+
+ Flags are numbered from the least-significant bit, at bit 0 (i.e. 0x1,
+ an _even_ bit). They are generally assigned in pairs so that features
+ can be introduced as optional (_odd_ bits) and later upgraded to be compulsory
+ (_even_ bits), which will be refused by outdated nodes:
+
+ CONTEXT:
+ * `I`: presented in the `init` message.
+ * `N`: presented in the `node_announcement` messages
+ * `C`: presented in the `channel_announcement` message.
+ * `C-`: presented in the `channel_announcement` message, but always odd (optional).
+ * `C+`: presented in the `channel_announcement` message, but always even (required).
+ * `9`: presented in [BOLT 11](11-payment-encoding.md) invoices.
+
+    FEATURE_NAME                          # CONTEXT
+    ---------------------------------------------- """
+    OPTION_DATA_LOSS_PROTECT = 0          # IN
+    INITIAL_ROUTING_SYNC = 2              # I
+    OPTION_UPFRONT_SHUTDOWN_SCRIPT = 4    # IN
+    GOSSIP_QUERIES = 6                    # IN
+    VAR_ONION_OPTIN = 8                   # IN9
+    GOSSIP_QUERIES_EX = 10                # IN
+    OPTION_STATIC_REMOTEKEY = 12          # IN
+    PAYMENT_SECRET = 14                   # IN9
+    BASIC_MPP = 16                        # IN9
+    OPTION_SUPPORT_LARGE_CHANNEL = 18     # IN
+    OPTION_ANCHOR_OUTPUTS = 20            # IN
+    OPTION_ANCHORS_ZERO_FEE_HTLC_TX = 22  # IN
+    OPTION_SHUTDOWN_ANYSEGWIT = 26        # IN
+    OPTION_CHANNEL_TYPE = 44              # IN
+    OPTION_SCID_ALIAS = 46                # IN
+    OPTION_PAYMENT_METADATA = 48          # 9
+    OPTION_ZEROCONF = 50                  # IN
```

Comment:
Do you want to add OPTION_PROPOSED_XXX for the others?  Or too much headache?

Reply:
You mean like from all the unmerged bolt PRs ?

- `option_dual_fund` ( PR 1009 ) 
- `option_closing_rejected` ( PR 1016 )
- `option_alternative_feerates` (PR 1036 )
- `option_attributable_error` ( PR 1044 )
- `option_upfront_fee` ( PR 1052 )